src/Sfa.Tl.ResultsAndCertification.Web/Loader/AdminDashboardLoader.cs

```diff
@@ -38,5 +38,12 @@ public async Task<T> GetAdminLearnerRecordAsync<T>(int pathwayId)
             return _mapper.Map<T>(response);
         }
 
+        public async Task<bool> ProcessChangeStartYearAsync(ReviewChangeStartYearViewModel reviewChangeStartYearViewModel)
+        {
+            var reviewChangeStartYearRequest = _mapper.Map<ReviewChangeStartYearRequest>(reviewChangeStartYearViewModel);
+            var response = await _internalApiClient.ProcessChangeStartYearAsync(reviewChangeStartYearRequest);
+            return _mapper.Map<bool>(response);
```

Comment:
if response is a bool why do we need to map it to a new bool? You should just return response